input/data/manure/refreshed_manure_management.json

```
@@ -11,5 +13,33 @@
                 "storage": 1.0
             }
         }
+    ],
+    "separator_configs":[
+        {
+            "name": "rotary_screen",
+            "processor_type": "separator",
+            "separator_type": "rotary screen",
+            "percent_dry_solids": 0.20,
```

```suggestion
            "separated_solids_dry_matter": 0.20,
```
Just a change to improve the clarity of what this variable represents

---

That's fine by me. What do you think about updating this name in the Separator attributes to match this (right now it's `percent_dry_solids`? I think it would be helpful if they matched since the rest do.

---

Very good point! We should definitely do that. I'm sorry to make this change at the inputs stage instead of in the actual separator relocation PR, I will try to note this sort of thing earlier going forward to we don't have to go back to already-merged files to change names